src/applications/virtual-agent/tests/components/webchat/startConvoAndTrackUtterances.unit.spec.js

```diff
@@ -367,5 +367,21 @@ describe('makeBotStartConvoAndTrackUtterances actions', () => {
       );
       expect(isRxSkillSessionStorageSet).to.equal(null);
     });
+    it('should pass isMobile to PVA', async () => {
+      await StartConvoAndTrackUtterances.makeBotStartConvoAndTrackUtterances(
+        'csrfToken',
+        'apiSession',
+        'apiURL',
+        'baseURL',
+        'userFirstName',
+        'userUuid',
+        'isMobile',
```

Comment:
Wouldn't this be a boolean value as opposed to a string?

Reply:
thats just a test that whatever we put in is passed through - we could have done a boolean, but the same test applies

Reply:
Its good practice to make the tests reflect the real data types of production code - this serves as documentation of how the code works on a deeper level.

Reply:
We have updated the code to reflect as such